### ISSUE TITLE
Lint can't follow `include`s?

### ISSUE THREAD

**Issue by NHDaly** [tonyhffong/Lint.jl#248]
Hi, I don't know if maybe my system is configured weird, but it seems like `Lint.jl` is unable to follow `include` statements. Therefore, when linting a file I get all sorts of "use of undeclared symbol" errors. This happens even when using `lintpkg`, which sounds like it's designed for this sort of thing.

For example, here I'm linting `Lint` itself:

```julia
julia> using Lint

julia> lintpkg("Lint")
>> /Users/daly/.julia/v0.6/Lint/src/statictype.jl:31 E321 EQ_METHOD_FALSE: use of undeclared symbol
/Users/daly/.julia/v0.6/Lint/src/statictype.jl:31 E321 EQ_METHOD_FALSE: use of undeclared symbol
/Users/daly/.julia/v0.6/Lint/src/statictype.jl:31 E321 EQ_METHOD_FALSE: use of undeclared symbol
/Users/daly/.julia/v0.6/Lint/src/statictype.jl:43 I382 f: argument declared but not used
/Users/daly/.julia/v0.6/Lint/src/statictype.jl:43 I382 argtypes: argument declared but not used
/Users/daly/.julia/v0.6/Lint/src/linttypes.jl:142 E321 msg: use of undeclared symbol
/Users/daly/.julia/v0.6/Lint/src/messages.jl:27 E321 level: use of undeclared symbol
/Users/daly/.julia/v0.6/Lint/src/messages.jl:27 E321 level: use of undeclared symbol
...
```
(there are almost 100 lines of output.)

------

Is this a bug? Or a known limitation?

It makes it pretty unusable for me, especially in Atom since so many lines are lit up with false positives.
